### ISSUE TITLE
Findmymac Module didn't load

### ISSUE THREAD

**Issue by mikaellofgren** [munkireport/munkireport-php#550]
Hi!
Couldn't get Findmy Mac to Module to get any data. Checked if its was loaded and get 404 error. 
Added this to findmymac_controller.php and it started to work again.
And Enabled should have green color instead of red.
  /**
    * Default method
	 *
	 **/
	function index()
	{
		echo "You've loaded the findmymac module!";
	}
  /**

**Comment by bochoven** [comment]
The module should run fine without the default method.

**Comment by mikaellofgren** [comment]
Thanks! It works fine now with 2.10.1.2197. But what about switching colors Green=Enabled, Red=Disabled?

**Comment by clburlison** [comment]
I personally disagree with your color switch request. This would be cool as one of the things to customize however it's super cosmetic. If you look at the purpose of munkireport and the target audience the current color scheme aligns correctly.

**Comment by mikaellofgren** [comment]
Yes its cosmetic. But have a look a Listing-> Network Enabled= Green, Security Encrypted= Green. With Filevault its switched...
